src/templates/next/components/UnorderedList/UnorderedList.tsx

```diff
@@ -1,28 +1,15 @@
 import type { UnorderedListProps } from "~/common"
-import OrderedList from "../OrderedList"
-import { BaseParagraph } from "../shared/Paragraph"
-import { Paragraph } from "../../typography/Paragraph"
+import ContentHole from "../../tiptap/ContentHole"
 
-const UnorderedList = ({ items }: UnorderedListProps) => {
+const UnorderedList = ({
+  content,
+  NodeViewWrapper = "ul",
+  NodeViewContent,
+}: UnorderedListProps) => {
   return (
-    <ul className="list-disc ps-8 mt-6">
```

Comment:
hmm ok just so my understanding is correct here. If I want to style the elements inside the content hole, I have to create a new element called `li` right? (or simply extend it from tiptap's implementation and add the classes that I require?)